### PR TITLE
Client: add get archived edges call

### PR DESCRIPTION
**Pull request by melisabok**

### CODE REVIEW

File: openreview/api/client.py [hunk 1]
```diff
@@ -1531,6 +1532,23 @@ def get_grouped_edges(self, invitation=None, head=None, tail=None, label=None, g
         json = response.json()
         return json['groupedEdges'] # a list of JSON objects holding information about an edge
 
+    def get_edges_archive(self, invitation):
```
Comment (celestemartinez) [inline openreview/api/client.py:1535]: could we call this `get_archived_edges()`. I think it's a bit more descriptive

Comment (carlosmondra) [reply openreview/api/client.py:1535]: I agree

Comment (carlosmondra) [inline openreview/api/client.py:1535]:
```suggestion
    def get_archived_edges(self, invitation):
```

Comment (melisabok) [reply openreview/api/client.py]: Done!

**Review by celestemartinez — APPROVED** [review]

**Review by carlosmondra — APPROVED** [review]

**Pull request merged**